ThreeBotPackages/tfgrid_solutions/tfgrid_solutions/chatflows/your_flist.py

```diff
@@ -51,17 +51,19 @@ def flist_url(self):
         )
 
         self.user_form_data["Flist link"] = self.user_form_data["Flist link"].strip()
-        if ".md5" in self.user_form_data["Flist link"] or ".md" in self.user_form_data["Flist link"]:
-            self.user_form_data["Flist link"] = self.user_form_data["Flist link"].replace(".md5", "")
-            self.user_form_data["Flist link"] = self.user_form_data["Flist link"].replace(".md", "")
 
-        if "hub.grid.tf" not in self.user_form_data["Flist link"]:
+        if (
+            "hub.grid.tf" not in self.user_form_data["Flist link"]
+            or ".md" in self.user_form_data["Flist link"]
```

Comment:
plz remove the 2 or cases about md and md5
a url could always be 
https://hub.grid.tf/myuser.3bot/somproejct.md.flist